core/src/components/AppMenu.vue

```diff
@@ -20,7 +20,8 @@
   -->
 
 <template>
-	<nav class="app-menu">
+	<nav class="app-menu"
+		:aria-label="t('core', 'App menu')">
```

Comment:
Not sure if "app menu" is understandable

```suggestion
	<nav class="app-menu" :aria-label="t('core', 'Applications menu')">
```